Userland/Games/ColorLines/Marble.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Point.h>
+
+class Marble final {
+public:
+    using Point = Gfx::IntPoint;
+    using Color = u8;
```

Comment:
Maybe this should just rather be an `enum`? And have a `__Count` member in place of `number_of_colors`?

Reply:
I did this initially but then realized i do not need the individual color names.  I will add tow constants - one for number of colors and the empty cell constant instead.